code-challenges.test.js

```diff
@@ -44,3 +105,29 @@ const cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
 // b) Create the function that makes the test pass.
+
+
+describe("mySum", () => {
+  it("returns the sum of all the numbers cubed", () => {
+    console.log(mySum(cubeAndSum1))
+    expect(mySum(cubeAndSum1)).toEqual(99)
+    expect(mySum(cubeAndSum2)).toEqual(1125)
+  
+  })
+})
+// ReferenceError: mySum is not defined
+
+// Pseudocode : 1. Create a function called mySum
+//              2. Input takes in an array of numbers
+//              3. Use .reduce to return the sum of all the numbers in the array
+//              4  Use Math.pow() to return the value to a power, in this case since its cubed the 3rd power. The zero is where it starts in the iteration for .reduce
```

Comment:
Is the zero where it starts the iteration or the initial value that we will add to?